doc/source/reference/arrays.promotion.rst

```diff
@@ -99,32 +113,29 @@ In this case, an error is raised for integers:
   Traceback (most recent call last):
   ...
   OverflowError: Python int too large to convert to C long
-
-And overflows are possible for floating points:
-
   >>> np.float32(1) + 1e300
   np.float32(inf)
   ... RuntimeWarning: overflow encountered in cast
```

Comment:
Need to figure out how to make the doctest work with warnings... Or maybe just not make this a doc example at all.